.github/workflows/screener-run.yml

```diff
@@ -1,32 +1,49 @@
 name: Screener run
-
 on:
-  workflow_run:
-    workflows:
-      - Screener build
-    types:
-      - completed
+  pull_request:
```

Comment:
for testing purposes the workflow is triggered by PRs, but it will be changed to be triggered by the `workflow_run` event so that it will start running after the completion of the `screener-build.yml` workflow